components/user_settings/advanced/user_settings_advanced.jsx

```diff
@@ -47,6 +50,11 @@ export default class AdvancedSettingsDisplay extends React.Component {
                 'join_leave',
                 'true'
             ),
+            deactivate_account: PreferenceStore.get(
```

Comment:
Is it necessary to have/save this on user's preference store?

Reply:
you right dont need it

Reply:
@saturninoabril did the changes and rebased. thanks! :)